application/django/README.md

```diff
@@ -0,0 +1,24 @@
+Init the SecretHub demo repo with example values
+```
+secrethub demo init
+```
+
+Create a service account for the demo repo
+```
+secrethub service init --description demo_service \
+--permission read --file demo_service.cred <your_username>/demo
```

Comment:
Since we're working in a shell anyway, we could also replace `<your_username>` with `${SECRETHUB_USERNAME}` and let the user set it once after `secrethub demo init`. 

@florisvdg what do you think about this?

Reply:
Or what would you think about not using a service account, but a personal account? Password prompt works fine with `docker run`.

Reply:
"Why do I have to create a new account just to run some example code...?"

Reply:
That would also work. But then the comment is still valid for the next command.

Reply:
I'll add it in the README.md